backend/repo/pg/auth.go

```diff
@@ -39,6 +39,7 @@ func (r *AuthRepo) GetAuthUserinfoByIDs(ctx context.Context, authIDs []uint) (ma
 	err := r.db.WithContext(ctx).Table("auths").
 		Select("id,user_info as auth_user_info").
 		Where("id IN (?) ", authIDs).
+		Where("source_type NOT IN (?)", consts.BotSourceTypes).
 		Find(&authUserInfo).Error
```

Comment:
在查询认证用户信息时排除Bot类型的认证源，这是一个合理的改进。不过，为了提高代码的可读性和可维护性，建议添加注释说明为什么要排除这些Bot类型的认证源。

```suggestion
err := r.db.WithContext(ctx).Table("auths").
		Select("id,user_info as auth_user_info").
		Where("id IN (?) ", authIDs).
		// 排除Bot类型的认证源，因为Bot用户不需要显示用户信息
		Where("source_type NOT IN (?)", consts.BotSourceTypes).
		Find(&authUserInfo).Error